### PR TITLE
Add "happy" to apps list

### PR DESCRIPTION
**Pull request by zaceno**
"happy" is a happiness index calculator built with hyperapp. It's a quite simple app, esp data-wise. But it *is* an example of an app being used in the real-world.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ A curated list of awesome projects built with [HyperApp](https://github.com/hype
 
 ## Apps and Boilerplates
 
+- [happy](https://github.com/zaceno/happy) - Happiness index calculator for teams, built with hyperapp.
```
Comment (tobiasbueschel) [inline README.md]: ~hyperapp~ = HyperApp

Comment (zaceno) [inline README.md]: Good catch! Fixed in new commit.

**Comment by zaceno** [comment]
@lukejacksonn : be my guest! I honestly didn't spend a lot of time on looks.

**Comment by zaceno** [comment]
Is anything holding this up? @tobiasbueschel requested a change that I fixed. @jbucaran said 👍 . @lukejacksonn wants to style the app, which he may but I doubt he meant it needs to look prettier to be on the list (or?)

Not that I'm in a rush at all. Just unsure if I'm supposed to do anything more ;)

**Comment by zaceno** [comment]
🎉

**Comment by tobiasbueschel** [comment]
👍 

Sorry for the little delay @zaceno 

Cheers :)

**Comment by zaceno** [comment]
No problem :)

**Pull request merged**